static/app/components/dropdownMenuControl.tsx

```diff
@@ -8,9 +8,9 @@ import {Item, Section} from '@react-stately/collections';
 import {useMenuTriggerState} from '@react-stately/menu';
 import {MenuTriggerProps} from '@react-types/menu';
 
-import DropdownButton, {DropdownButtonProps} from 'sentry/components/dropdownButtonV2';
-import {MenuItemProps} from 'sentry/components/dropdownMenuItemV2';
-import Menu from 'sentry/components/dropdownMenuV2';
+import DropdownButton, {DropdownButtonProps} from 'sentry/components/dropdownButton';
```

Comment:
It would be easier for you and reviewers if this were split into two PRs (one to rename deprecated component and one to rename v2).